test/vue-frag.spec.ts

```diff
@@ -1184,3 +1187,40 @@ test('nested fragments', async () => {
 	`);
 	wrapper.expectMatchingDom();
 });
+
+// #67
+test('Set innerHTML of empty fragment', async () => {
+	const usage = {
+		template: `
+        <div>
+          <div v-frag ref="fragment" />
+        </div>
+        `,
+		directives: { frag },
+		data() {
+			return {
+				html: '',
+			};
+		},
+		watch: {
+			html() {
+				this.$refs.fragment.innerHTML = this.html;
```

Comment:
This isn't a use-case I'd like to support.

If users directly manipulate the `innerHTML` of the fragment, it's bound to break one way or another.

Reply:
>If users directly manipulate the DOM, it's bound to break one way or another.

Agree, direct DOM manipulation using ref is for testing.
So, it would be better to change the name to something like `forTest` instead of `usage` so as not to misunderstand it as a use-case.

>Is this how you encountered this bug? Or was it a different way?

I didn't actually use refs to manipulate the DOM.
I found the bug when I used a custom directive to set innerHTML.

Reply:
So, to clarify, this bug not reproducible without directly manipulating the DOM?

Reply:
Yes, at least as far as I know.
It's not reproducible without directly manipulating the DOM.
(using a directive like :html, using ref like test code, etc)